doc/topics/development/contributing.rst

```diff
@@ -219,201 +219,34 @@ Fork a Repo Guide_>`_ and is well worth reading.
 
-.. _backporting-pull-requests:
-
-Backporting Pull Requests
-=========================
-
-If a bug is fixed on ``|repo_primary_branch|`` and the bug is also present on a
-currently-supported release branch, it will need to be back-ported to an
-applicable branch.
-
-.. note:: Most Salt contributors can skip these instructions
-
-    These instructions do not need to be read in order to contribute to the
-    Salt project! The SaltStack team will back-port fixes on behalf of
-    contributors in order to keep the contribution process easy.
-
-    These instructions are intended for frequent Salt contributors, advanced
-    Git users, SaltStack employees, or independent souls who wish to back-port
-    changes themselves.
-
-It is often easiest to fix a bug on the oldest supported release branch and
-then merge that branch forward into ``|repo_primary_branch|`` (as described earlier in this
-document). When that is not possible the fix must be back-ported, or copied,
-into any other affected branches.
-
-These steps assume a pull request ``#1234`` has been merged into ``|repo_primary_branch|``.
-And ``upstream`` is the name of the remote pointing to the main Salt repo.
-
-#.  Identify the oldest supported release branch that is affected by the bug.
-
-#.  Create a new branch for the back-port by reusing the same branch from the
-    original pull request.
-
-    Name the branch ``bp-<NNNN>`` and use the number of the original pull
-    request.
-
-    .. code-block:: bash
-
-        git fetch upstream refs/pull/1234/head:bp-1234
-        git checkout bp-1234
-
-#.  Find the parent commit of the original pull request.
-
-    The parent commit of the original pull request must be known in order to
-    rebase onto a release branch. The easiest way to find this is on GitHub.
-
-    Open the original pull request on GitHub and find the first commit in the
-    list of commits. Select and copy the SHA for that commit. The parent of
-    that commit can be specified by appending ``~1`` to the end.
-
-#.  Rebase the new branch on top of the release branch.
-
-    * ``<release-branch>`` is the branch identified in step #1.
-
-    * ``<orig-base>`` is the SHA identified in step #3 -- don't forget to add
-      ``~1`` to the end!
-
-    .. code-block:: bash
-
-        git rebase --onto <release-branch> <orig-base> bp-1234
-
-    Note, release branches prior to ``2016.11`` will not be able to make use of
-    rebase and must use cherry-picking instead.
-
-#.  Push the back-port branch to GitHub and open a new pull request.
-
-    Opening a pull request for the back-port allows for the test suite and
-    normal code-review process.
-
-    .. code-block:: bash
+Handling CVE’s
+--------------
 
-        git push -u origin bp-1234
+Salt will make a new release identical to its last.
+The only difference will be the path/fix for the CVE.
+This should make the upgrade process a lot smoother for people
+because the odds of something breaking is a lot smaller.
```

Comment:
Minor revisions to the last sentence to make it a little cleaner:

This method should improve the upgrade process by reducing the chances of breaking something.

NOTE: Instead of "improve" you could also say "streamline" if you like that better.